skt/executable.py

```diff
@@ -421,10 +421,15 @@ def cmd_run(cfg):
                                         cfg.get('wait'),
                                         arch=cfg.get("kernel_arch"))
 
-    for index, job in enumerate(runner.job_to_recipe_map.keys()):
+    recipe_set_index = 0
```

Comment:
I get that these changes are hard to split, I know how that happens. I'm willing to accept it as is, to speed you up, but I would not be able to say I did *any* meaningful review of this, because I just don't have the energy nor time to understand it as is. Perhaps @major would be able to do it :)